namespaces/ci_job.yml

```diff
@@ -47,6 +47,21 @@ namespace:
     description: >
       Full URL to the CI job on the CI master.
 
+  - name: node_name
```

Comment:
the hostname is the Jenkins master. While the node_name shows which node of Jenkins master execute the job.